airflow/operators/bash.py

```diff
@@ -141,14 +142,18 @@ def __init__(
         append_env: bool = False,
         output_encoding: str = "utf-8",
         skip_exit_code: int = 99,
```

Comment:
Wouldn’t this always trigger the deprecation warning? We need to change the default `skip_exit_code` to None.

Reply:
+1